userspace/falco/stats_writer.cpp

```diff
@@ -309,6 +309,77 @@ stats_writer::collector::collector(const std::shared_ptr<stats_writer>& writer)
 {
 }
 
+std::string merge_strings(const std::vector<std::string>& v, const std::string& delim) {
```

Comment:
I believe we have utils somewhere to do this. We do have `sinsp_join()`, maybe we could use it here? Can you check if it works fine for your purpose?

Reply:
Yes I will check and update